### DISCUSSION TITLE
Multi-page popups (kinda like the Options panel)

### DISCUSSION THREAD

**Discussion by TechStudent10** [geode-sdk/sdk#101] [Help] [answered]
How might one create a popup with multiple pages in Geode.

Example: `options` (not my image fyi, taken from Google 'cuz I'm too lazy rn to open GD)

![image](https://user-images.githubusercontent.com/76978184/208219777-cf5761e0-d740-4141-9ab1-218c64cd338f.png)

**Comment by Djspac3** [comment]
Wouldn't this just be a button on the popup that opens the next page as a new popup? (Overwriting 1st popup)

**Reply by TechStudent10** [reply, marked as answer]
Yep

Also this discussion is over a year old so like

**Reply by Djspac3** [reply]
holy jee

Edit lol u got it as answer just for conferring what I was thinkin
